commands/gen/gen_dao.go

```diff
@@ -73,6 +75,7 @@ OPTION
                          | Kebab           | any-kind-of-string |
                          | KebabScreaming  | ANY-KIND-OF-STRING |
     -/--stdTime          use time.Time from stdlib instead of gtime.Time for generated time/date fields of tables.
+	-/--jsonSupport      use jsonSupport to use *gjson.Json instead of string for geerated json fields of tables.
```

Comment:
1. `geerated` -> `generated`
2. Use spaces instead of indent character in the help content.

Reply:
Fixed